src/app/components/ms-item/ms-item.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="section" [class.has-nested]="nested" [class.has-nested2]="nested2">   
+    <div class="row no-gutters" *ngIf="data?.locus">
+        <span class="col-1" *ngIf="nested || nested2"></span>
+        <span class="col-1" *ngIf="nested2"></span>
+        <div class="col-1 col-label">
+            <span *ngIf="data?.locus" class="locus">
+                <evt-content-viewer [content]="data?.locus"></evt-content-viewer>
+                <span>:</span>
+            </span>
+        </div>
+        <div class="col-6 col-content">
+            <span *ngFor="let t of data.title | keyvalue">
+                <evt-content-viewer [content]="data.title[t.key]" class="title"></evt-content-viewer>
+                <span *ngIf="t.key < [data.title.length-1]">, </span>
+            </span>
+            <span *ngFor="let dt of data.docTitle | keyvalue">
+                <evt-content-viewer [content]="data.docTitle[dt.key]" class="title"></evt-content-viewer>
+                <span *ngIf="dt.key < [data.docTitle.length-1]">, </span>
+            </span>
+            <evt-ms-desc-section *ngIf="data?.textLang.length>0" [label]="'textLang'" [inlineLabel]="true">
+                <div content>
+                    <span *ngFor="let tl of data.textLang | keyvalue">
+                        <evt-content-viewer [content]="data.textLang[tl.key]"></evt-content-viewer>
+                        <span *ngIf="tl.key < [data.textLang.length-1]">, </span>
+                    </span>
+                </div>
+            </evt-ms-desc-section>
+            <evt-ms-desc-section *ngIf="data?.author.length>0 || data?.docAuthor.length>0" [label]="'author'" [inlineLabel]="true">
+                <div content>
+                    <span *ngFor="let a of data.author | keyvalue">
+                        <evt-content-viewer [content]="data.author[a.key]"></evt-content-viewer>
+                        <span *ngIf="a.key < [data.author.length-1]">, </span>
+                    </span>
+                    <span *ngFor="let da of data.docAuthor | keyvalue">
+                        <evt-content-viewer [content]="data.docAuthor[da.key]"></evt-content-viewer>
+                        <span *ngIf="da.key < [data.docAuthor.length-1]">, </span>
+                    </span>
+                </div>
+            </evt-ms-desc-section>
+            <evt-ms-desc-section *ngIf="data?.docDate.length>0" [label]="'docDate'" [inlineLabel]="true">
+                <evt-content-viewer *ngFor="let dd of data.docDate" [content]="dd" content></evt-content-viewer>
+            </evt-ms-desc-section>
+            <evt-ms-desc-section *ngIf="data?.colophon.length>0" [label]="'colophon'" [inlineLabel]="true">
```

Comment:
If you check the mockup, the nested msItem label-value pair is rendered differently. I'm not sure about the layout of msDescSection component. Maybe you can optimize it for using in different cases. Think about it.